libsweep/include/Error.hpp

```diff
@@ -0,0 +1,12 @@
+#ifndef SWEEP_ERROR_2EADE195E243_H
+#define SWEEP_ERROR_2EADE195E243_H
+
+#include <stdexcept>
+
+namespace sweep {
+    struct ErrorBase : std::runtime_error {
```

Comment:
Can you please lowercase the file to `error.h` as well as `error` here: it's already in the `sweep` namespace, so it's the generic `sweep::error` type.